src/main/java/study/StringCalculator.java

```diff
@@ -0,0 +1,69 @@
+package study;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    public int calculate(String input) {
+        if (isNullOrBlank(input)) {
+            return 0;
+        }
+
+        Matcher matcher = getMatcher(input);
+
+        if(matcher.find()){
+            return sumArrayValue(stringToInt(splitByPatten(matcher)));
+        }
+
+        return sumArrayValue(stringToInt(split(input)));
+    }
+
+    private Matcher getMatcher(String input) {
+        Matcher matcher = Pattern.compile("//(.)\n(.*)").matcher(input);
+        return matcher;
+    }
```

Comment:
로컬 변수를 별도로 활용하지 않는다면 바로 리턴해도 괜찮을 것 같아요
```suggestion
    private Matcher getMatcher(String input) {
        return Pattern.compile("//(.)\n(.*)").matcher(input);
    }
```